src/main/java/com/frameworkium/jira/zapi/StepExecution.java

```diff
@@ -0,0 +1,130 @@
+package com.frameworkium.jira.zapi;
```

Comment:
Can you swap all the logic of null checking to if (variable == null) rather than (null == variable)?